capstone/src/main/java/com/google/sps/CloudNLPAssociation.java

```diff
@@ -73,9 +75,9 @@ public CloudNLPAssociation(LanguageServiceClient language) {
    * @param messages the open text to be analyzed
    * @return an arraylist of all the entity sentiments in the messages
    */
-  public ArrayList<EntitySentiment> analyzeAssociations(ArrayList<String> messages) {
+  public ArrayList<EntitySentiment> analyzeAssociations(ArrayList<AssociationInput> messages) {
     ArrayList<EntitySentiment> res = new ArrayList<EntitySentiment>();
-    for (String message : messages) {
+    for (AssociationInput message : messages) {
```

Comment:
I think it was really cool to make a class instead of continuing to use the string class!

Reply:
Fixed!